Examples.fshx

```diff
@@ -53,32 +133,154 @@ Instance: mCODEKarnofskyPerformanceStatusExample01
+* category = SCT#53438000 "Radiation therapy procedure or service (procedure)"
+* code = SCT#152198000 "Brachytherapy (procedure)"
+* subject = Reference(mCODEPatientExample01)
+* asserter = Reference(mCODEPractitionerExample01)
+* performedDateTime = "2019-03-01"
+* extension[TreatmentIntent].valueCodeableConcept = SCT#373808002 "Curative - procedure intent"
+* extension[RadiationDose].extension[totalRadiationDoseDelivered].valueQuantity.code = UCUM#cGy 
+* extension[RadiationDose].extension[totalRadiationDoseDelivered].valueQuantity.value = "1200"
+* reasonReference = Reference(mCODEPrimaryCancerConditionExample01)
+* bodySite = SCT#41224006 "Structure of lower lobe of left lung (body structure)"
+
+
+Instance: mCODETNMClinicalDistantMetastasesCategoryExample01
+InstanceOf: TNMClinicalDistantMetastasesCategory
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/TNMClinicalDistantMetastasesCategory"
+* status = #final "final" 
+* category = #laboratory "laboratory"
+* method = MTH#C146985 "AJCC Cancer Staging Manual 8th Edition" 
+* code = LNC#21907-1 "Distant metastases.clinical [Class] Cancer"
+* subject = Reference(mCODEPatientExample01)
+* effectiveDateTime = "2019-04-01"
+* valueCodeableConcept = AJCC#cM0 "M0"
+
+
+Instance: mCODETNMClinicalPrimaryTumorCategoryExample01
+InstanceOf: TNMClinicalPrimaryTumorCategory
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/TNMClinicalPrimaryTumorCategory"
+* status = #final "final" 
+* category = #laboratory "laboratory"
+* method = MTH#C146985 "AJCC Cancer Staging Manual 8th Edition" 
+* code = LNC#21905-5 "Primary tumor.clinical [Class] Cancer"
+* subject = Reference(mCODEPatientExample01)
+* effectiveDateTime = "2019-04-01"
+* valueCodeableConcept = AJCC#cT3 "T3"
+
+
+Instance: mCODETNMClinicalRegionalNodesCategoryExample01
+InstanceOf: TNMClinicalRegionalNodesCategory
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/TNMClinicalRegionalNodesCategory"
+* status = #final "final" 
+* category = #laboratory "laboratory"
+* method = MTH#C146985 "AJCC Cancer Staging Manual 8th Edition" 
+* code = LNC#21906-3 "Regional lymph nodes.clinical [Class] Cancer"
+* subject = Reference(mCODEPatientExample01)
+* effectiveDateTime = "2019-04-01"
+* valueCodeableConcept = AJCC#cN3 "N3"
+
+
+Instance: mCODETNMClinicalStageGroupExample01
+InstanceOf: TNMClinicalStageGroup
+* meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/TNMClinicalStageGroup"
+* status = #final "final" 
+* category = #laboratory "laboratory"
+* method = MTH#C146985 "AJCC Cancer Staging Manual 8th Edition" 
+* code = LNC#21908-9 "Stage group.clinical Cancer"
+* subject = Reference(mCODEPatientExample01)
+* effectiveDateTime = "2019-04-01"
+* valueCodeableConcept = AJCC#3C "IIIC"
+* hasMember = Reference(mCODETNMClinicalPrimaryTumorCategoryExample01) and Reference(mCODETNMClinicalRegionalNodesCategoryExample01) and Reference(mCODETNMClinicalDistantMetastasesCategoryExample01)
```

Comment:
Can't do this in one line. It should be:
* hasMember[TNMClinicalPrimaryTumorCategory] = Reference(mCODETNMClinicalPrimaryTumorCategoryExample01)
* hasMember[TNMClinicalRegionalNodesCategory] = Reference(mCODETNMClinicalRegionalNodesCategoryExample01) 
* hasMember[TNMClinicalDistantMetastasesCategory] = Reference(mCODETNMClinicalRegionalNodesCategoryExample01)